Formula/tsv-utils.rb

```diff
@@ -0,0 +1,19 @@
+class TsvUtils < Formula
+  desc "Command-line utilities for tabular data files (TSV)"
+  homepage "https://github.com/eBay/tsv-utils"
+  url "https://github.com/eBay/tsv-utils/archive/refs/tags/v2.2.1.tar.gz"
+  sha256 "daf6e88bff1843e69c066453ed384869eec0e497914a6a9c260791ef91d95241"
+  license "BSL-1.0"
+
+  depends_on "ldc" => :build
+  depends_on "make" => :build
+
+  def install
+    system "make", "DCOMPILER=ldc2", "LDC_PGO=2"
+    bin.install Dir["bin/*"]
+  end
+
+  test do
+    system "#{bin}/tsv-select", "--help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.